packages/widgetbook/lib/src/app_info/app_info.dart

```diff
@@ -1,9 +1,2 @@
-class AppInfo {
-  AppInfo({
-    required this.name,
-  });
-
-  /// The name of the app or project for which the Widgetbook is created.
-  /// This name will be displayed in the upper left corner.
-  final String name;
-}
+export './models/models.dart';
+export './providers/providers.dart';
```

Comment:
The problem I had before was that this file was exported from `src/models/models.dart` even though it wasn't in that folder. So I tried to create a folder structure such that `AppInfo` would be a separate feature but I understand now that it will not be needed. Should I delete the usage of `AppInfo` in `widgetbook`?